### PR TITLE
3 New Regex Tutorial Links in Useful-Links

### PR DESCRIPTION
**Pull request by rpanczer**
One overview, one interactive exercises and one to the python 3 regex syntax.

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by rpanczer** [comment]
I am not sure why review/gitmate/pr keeps failing. I fixed the EOL issue, bad word 'Added' issue. The most recent gitmate-bot fail does not contain any details. Any ideas why this is failing? Am I missing something obvious?

**Comment by rpanczer** [comment]
@Makman2 @khanchi97 Can you guys take a look at this PR when you get a chance? Thanks

**Comment by CCInCharge** [comment]
Just add a space between the colon and the next word, e.g.
docs: Add regex ............

**Comment by CCInCharge** [comment]
ack 9420d15

**Comment by rpanczer** [comment]
@CCInCharge thanks for the review! Is there anything I have to do for this to be merged? First PR.

**Comment by CCInCharge** [comment]
@rpanczer no worries! One of the owners will take a look and merge the PR when ready, or leave comments if they see anything else. I'm not one of the owners, just a fellow community member like you, but it helps save them time when we review each other's PRs. :)

File: docs/Developers/Useful_Links.rst [hunk 1]
```diff
@@ -44,3 +44,11 @@ coala-Links
...
 -   :doc:`coala Newcomers' Guide<Newcomers_Guide>`
 -   :doc:`coala Review Process<Review>`
+
+Regex-Links
+-----------
+
+-   `Regex Tutorial/Overview <https://ryanstutorials.net/regular-expressions-tutorial/>`_
+-   `Regex Interactive Exercises <https://regexone.com/>`_
+-   `Regex Python Syntax <https://docs.python.org/3.6/library/re.html>`_
+
```
Comment (Makman2) [inline docs/Developers/Useful_Links.rst]: unnecessary newline ;)

File: docs/Developers/Useful_Links.rst [hunk 2]
```diff
@@ -44,3 +44,11 @@ coala-Links
 -   `coala Chat <https://gitter.im/coala/coala>`_
 -   :doc:`coala Newcomers' Guide<Newcomers_Guide>`
 -   :doc:`coala Review Process<Review>`
+
+Regex-Links
+-----------
+
+-   `Regex Tutorial/Overview <https://ryanstutorials.net/regular-expressions-tutorial/>`_
+-   `Regex Interactive Exercises <https://regexone.com/>`_
+-   `Regex Python Syntax <https://docs.python.org/3.6/library/re.html>`_
```
Comment (Makman2) [inline docs/Developers/Useful_Links.rst]: hmm regex doesn't have its own "syntax"... Maybe ~~Syntax~~ --> Module?

Comment (Makman2) [reply docs/Developers/Useful_Links.rst]: or even
`Python Regex Module Documentation`

File: docs/Developers/Useful_Links.rst [hunk 3]
```diff
@@ -44,3 +44,11 @@ coala-Links
 -   `coala Chat <https://gitter.im/coala/coala>`_
 -   :doc:`coala Newcomers' Guide<Newcomers_Guide>`
 -   :doc:`coala Review Process<Review>`
+
+Regex-Links
+-----------
+
+-   `Regex Tutorial/Overview <https://ryanstutorials.net/regular-expressions-tutorial/>`_
```
Comment (Makman2) [inline docs/Developers/Useful_Links.rst]: --> `Regex Overview/Tutorial`
Overview is more general than a tutorial, I think from reading-flow perspective it should come first :+1:

**Comment by Makman2** [comment]
unack 38f5b78

**Comment by Makman2** [comment]
unack 9420d15

**Comment by Makman2** [comment]
ack 2867e7e

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**